docs/06-api-lifecycle/01-duration-terms.adoc

```diff
@@ -1,29 +1,20 @@
 // tag::DE[]
 |===
-| Dauer: XXX Min. | Übungszeit: XXX Min.
+| Dauer: 90 Min. | Übungszeit: 60 Min.
 |===
 
 === Begriffe und Konzepte
-Begriff 1, Begriff 2, Begriff 3
+
+API Lifecycle, API Produkt, Linting, Consumer-driven Contract Testing, API Gateway
```

Comment:
Sollen wir "Consumer-driven" rausschmeissen und einfach nur "Contract Testing"?

Reply:
Oder vielleicht "Consumer-driven" in Klammern?

Reply:
Ok, wenn das hilft auch gut. Je weniger wir uns festlegen auf spezielle Nischen desto besser.

Reply:
Ja, finde ich gut. Passe ich noch an.